demo/src/app/components/tabset/demos/index.ts

```diff
@@ -1,9 +1,4 @@
-import {NgbdTabsetBasic} from './basic/tabset-basic';
-
-export const DEMO_DIRECTIVES = [NgbdTabsetBasic];
-
-export const DEMO_SNIPPETS = {
-  "basic": {
-    "code": require('!!prismjs?lang=typescript!./basic/tabset-basic'),
-    "markup": require('!!prismjs?lang=markup!./basic/tabset-basic.html')}
-};
```

Comment:
This change is not good. Please add your imports from code / markup here instead of putting them in index.ts of individual demos.